### PR TITLE
106 to reate table

### PR DESCRIPTION
**Pull request by Kyslytsia**

### CODE REVIEW

**Review by ASGladchenko — APPROVED** [review]

File: client/tailwind.config.ts [hunk 1]
```diff
@@ -39,7 +39,11 @@ const darkGray = '#49454F';
 const blueCrayolaToDeepBlue = `linear-gradient(135deg, ${blueCrayola} 0%, ${deepBlue} 100%)`;
 
 const config: Config = {
-  content: ['./src/components/**/*.{js,ts,jsx,tsx,mdx}', './src/app/**/*.{js,ts,jsx,tsx,mdx}'],
+  content: [
+    './src/components/**/*.{js,ts,jsx,tsx,mdx}',
+    './src/pages/**/*.{js,ts,jsx,tsx,mdx}',
```
Comment (Wolshebnik) [inline client/tailwind.config.ts:44]: './src/pages/**/*.{js,ts,jsx,tsx,mdx}'
 remove pls
it not need anymore

File: client/src/components/pages/table-requests/mock.ts [hunk 2]
```diff
@@ -107,3 +106,5 @@ export const data: Organization[] = [
     email: 'qZjzA@example.com',
   },
 ];
+
+export default data;
```
Comment (Wolshebnik) [inline client/src/components/pages/table-requests/mock.ts:110]: why default export?

File: client/src/components/pages/table-requests/index.tsx [hunk 3]
```diff
@@ -67,4 +67,4 @@ const TableRequests = () => {
   );
 };
 
-export { TableRequests };
+export default TableRequests;
```
Comment (Wolshebnik) [inline client/src/components/pages/table-requests/index.tsx:70]: Why default export

File: client/src/components/pages/table-organization/row-item.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </>
+  );
+};
+
+export default RowItem;
```
Comment (Wolshebnik) [inline client/src/components/pages/table-organization/row-item.tsx:94]: Why default export

File: client/src/components/pages/table-organization/mock.ts [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    organizationName: 'Назва Організації',
+    EDRPOU: 12345678,
+    user: 1,
+    date: '22.02.2023',
+    doc: 'https://www.w3schools.com/w3images/lights.jpg',
+    email: 'qZjzA@example.com',
+  },
+];
+
+export default data;
```
Comment (Wolshebnik) [inline client/src/components/pages/table-organization/mock.ts:124]: why default export?

File: client/src/components/pages/table-organization/index.tsx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+        current={page}
+        pageSize={100}
+        onChange={setPage}
+        className="py-16 max-w-[440px] my-auto desktop:ml-11"
+      />
+    </div>
+  );
+};
+
+export default TableOrganization;
```
Comment (Wolshebnik) [inline client/src/components/pages/table-organization/index.tsx:72]: Why default export?

File: client/src/app/[locale]/(admin)/admin/requests/page.tsx [hunk 7]
```diff
@@ -1,6 +1,5 @@
 import { Metadata } from 'next';
-
-import { TableRequests } from '@/components';
+import TableRequests from '@/components/pages/table-requests';
```
Comment (Wolshebnik) [inline client/src/app/[locale]/(admin)/admin/requests/page.tsx:2]: use import from just @/components

File: client/src/app/[locale]/(admin)/admin/organizations/page.tsx [hunk 8]
```diff
@@ -1,3 +1,4 @@
+import TableOrganization from '@/components/pages/table-organization';
```
Comment (Wolshebnik) [inline client/src/app/[locale]/(admin)/admin/organizations/page.tsx:1]: use import from just @/components

File: client/src/app/[locale]/(admin)/admin/declined/page.tsx [hunk 9]
```diff
@@ -1,6 +1,5 @@
 import { Metadata } from 'next';
-
-import { TableRequests } from '@/components';
+import TableRequests from '@/components/pages/table-requests';
```
Comment (Wolshebnik) [inline client/src/app/[locale]/(admin)/admin/declined/page.tsx:2]: use import from just @/components
and sort

**Review by Wolshebnik — CHANGES_REQUESTED** [review]

**Pull request merged**